docs/use-cases/tokenization/real-world-assets.page.tsx

```diff
@@ -0,0 +1,431 @@
+import { useThemeHooks } from "@redocly/theme/core/hooks";
+import { Link } from "@redocly/theme/components/Link/Link";
+
+export const frontmatter = {
+  seo: {
+    title: "RWA Tokenization on XRP Ledger",
+    description:
+      "Power blockchain tokenization use cases on the XRP Ledger. Learn how to build RWA tokenization solutions with developer tools and APIs. Get started today.",
+  },
+};
+
+const benefitsData = [
+  {
+    iconClass: "low-fees",
+    title: "Fast Settlement and Low Fees",
+    description:
+      "Leverage transaction finality in 3-5 seconds at fractions of a cent, making XRP Ledger an ideal platform for handling large-scale tokenization with cost-effective, high-speed transactions.",
+  },
+  {
+    iconClass: "access",
+    title: "Access to Liquidity",
+    description:
+      "Integrate XRP Ledger's native liquidity pools and order books directly into your applications via API. Liquidity is aggregated at the protocol layer, rather than fragmented across various applications, providing developers with seamless access at the core blockchain level.",
+  },
+  {
+    iconClass: "full-stack",
+    title: "Full-Stack Tokenization Tools",
+    description:
+      "Issue, manage, and trade real-world assets without needing to build smart contracts. XRP Ledger's built-in functionality and compliance-enabling features allows you to focus on building without additional layers of complexity.",
+  },
+  {
+    iconClass: "best-in-class",
+    title: "Best-in-Class On-chain DEX",
+    description:
+      "Utilize XRP Ledger's native decentralized exchange (DEX) with integrated Automated Market Makers (AMM) and on-chain 24/7 order books, providing a developer-friendly environment to create DeFi solutions for traditional finance applications.",
+  },
+  {
+    iconClass: "cross-chain",
+    title: "Cross-Chain Interoperability*",
+    description:
+      "Enable seamless interaction with other blockchains via secure cross-chain bridges, allowing you to expand your project's functionality and liquidity options across ecosystems.",
+  },
+];
+const companies = [
+  {
+    className: "open-eden",
+    alt: "Open Eden",
+    aspectRatio: 5,
+    link: "https://openeden.com/tbill",
+  },
+  {
+    className: "zoniqx",
+    alt: "Zoniqx",
+    aspectRatio: 2.7,
+    link: "https://www.zoniqx.com/",
+  },
+  {
+    className: "archax",
+    alt: "Archax",
+    aspectRatio: 1.25,
+    link: "https://archax.com/",
+  },
+  {
+    className: "meld",
+    alt: "Meld",
+    aspectRatio: 0.81,
+    link: "https://www.meld.gold/",
+  },
+  {
+    className: "ripple-logo",
+    alt: "Ripple",
+    aspectRatio: 3.89,
+    link: "https://ripple.com/rwa-tokenization",
+  },
+];
+
+const features = [
+  {
+    title: "Token Issuance Tutorial",
+    link: "https://xrpl.org/docs/tutorials/how-tos/use-tokens/issue-a-fungible-token",
```

Comment:
Using a fully-specified URL here means that viewers of the translated site will get shunted to the English-language version of the site when they click it.

It's possible to make it work the intended way, but you need to do two things:

1. Use an absolute path here, e.g.

    ```suggestion
        link: "/docs/tutorials/how-tos/use-tokens/issue-a-fungible-token",
    ```
    (NOTE: If you use my commit below, it modifies these lines already so you don't need to use this suggestion separately.)
2. Change the `FeatureItem` definition below to use a `<Link>` element rather than an `onclick` handler.